CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-# NEXT RELEASE
```

Comment:
PS: sneaking this in, just noticed that the changelog is wrong on next major branch.